src/coreclr/jit/simdcodegenxarch.cpp

```diff
@@ -938,10 +938,10 @@ void CodeGen::genStoreLclTypeSIMD12(GenTree* treeNode)
 {
     assert((treeNode->OperGet() == GT_STORE_LCL_FLD) || (treeNode->OperGet() == GT_STORE_LCL_VAR));
 
+    treeNode->SetRegNum(REG_STK); // we always store the result on stack
```

Comment:
It seems odd to call this. Isn't the problem that LSRA shouldn't be assigning a register to this node at all?